e2e_test/sink/create_sink_as.slt

```diff
@@ -0,0 +1,29 @@
+statement ok
+CREATE TABLE t4 (v1 int, v2 int);
+
+statement ok
+CREATE MATERIALIZED VIEW mv4 AS SELECT * FROM t4;
```

Comment:
no need to add a mv here. sink can be directly after a table

Reply:
I just copied from other tests.